content/glance.pl.md

```diff
@@ -0,0 +1,109 @@
+---
+# Translation instructions are after the "#" character in this first section. They are comments that do not show up in the web page. You do not need to translate the instructions after "#".
+# In this first section, do not translate the words before a colon. For example, do not translate "title:". Do translate the text after "title:".
+
+title: "WCAG 2 w skrócie"
+nav_title: "W skrócie"
+description: Przedstawia sparafrazowane streszczenie Wytycznych dla dostępności treści internetowych (WCAG) 2.
+lang: pl  # Change "en" to the translated-language shortcode
+last_updated: 2024-02-25  # Put the date of this translation YYYY-MM-DD (with month in the middle)
+
+translators:
+  - name: "Stefan Wajda"
+github:
+  repository: w3c/wai-intro-wcag
+  path: 'content/glance.pl.md' # Add the language shortcode to the middle of the filename, for example: content/index.fr.md
+
+permalink: /standards-guidelines/wcag/glance/pl # Add the language shortcode to the end, with no slash at end, for example: /link/to/page/fr
+ref: /standards-guidelines/wcag/glance/ # Do not change this
+
+image: /content-images/wai-intro-wcag/general-social.jpg
+feedbackmail: wai@w3.org
+
+# In the footer below:
+# Do not change the dates
+# Translate the other words below, including "Date:" and "Editors:"
+# Translate the Working Group name. Leave the Working Group acronym in English.
+footer: >
+  <p><strong>Data:</strong> Updated 29 November 2023. First published July 2008.</p>
+  <p><strong>Redaktorzy:</strong> <a href="https://www.w3.org/People/Shawn">Shawn Lawton Henry</a> i Wayne Dick.</p>
+  <p>Opracowane z udziałem Grupy Roboczej ds. Edukacji i Kontaktów Zewnętrznych (<a href="https://www.w3.org/WAI/about/groups/eowg/">EOWG</a>) oraz Grupy Roboczej ds. wytycznych dla dostępności (<a href="https://www.w3.org/WAI/GL/">AG WG</a>).</p>
+
+# Przeczytaj uwagi dotyczące tłumaczenia tego zasobu na stronie https://github.com/w3c/wai-intro-wcag#readme
+---
+
+{::nomarkdown}
+{% include box.html type="start" h="2" title="Podsumowanie" class="full" %}
+{:/}
+
+Przedstawia sparafrazowane streszczenie Wytycznych dla dostępności treści internetowych 2.1 i 2.2.
+
+{::nomarkdown}
+{% include box.html type="end" %}
+{:/}
+
+Zapoznaj się z następującymi kluczowymi materiałami do nauki i korzystania z WCAG 2:
+-   **[[Omówienie WCAG]](/standards-guidelines/wcag/pl)**
+-   **[[Dokumenty WCAG 2]](/standards-guidelines/wcag/docs/pl)**
+
+Normatywne specyfikacje techniczne są na stronach [www.w3.org/TR/WCAG22](https://www.w3.org/TR/WCAG22/) (oryginalna wersja angielska) i [https://www.w3.org/Translations/WCAG21-pl/](https://www.w3.org/Translations/WCAG21-pl/) (autoryzowane tłumaczenie WCAG 2.1 na język polski)
+
+## Wytyczne
+
+{::nomarkdown}
+{% include box.html type="start" title="Postrzegalne" h=3 class="large" %}
```

Comment:
@zwiastunsw I have noticed the Guidelines names are different from those in the Authorized Translation. Is there any reason for this?

- Postrzegalne / Postrzegalność (WCAG 2.1 Authorized Translation)
- Funkcjonalne / Funkcjonalność (WCAG 2.1 Authorized Translation)
- Zrozumiałe / Zrozumiałość (WCAG 2.1 Authorized Translation)
- Solidne / Solidność (WCAG 2.1 Authorized Translation)